content/posts/newsletter/2021-09-05.md

```diff
@@ -0,0 +1,193 @@
+
+### Proposed
+
+The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.
+
+-   No XEPs proposed this month.
+
+### New
+
+-   Version 0.1.0 of [XEP-0460](https://xmpp.org/extensions/xep-0460.html) (Pubsub Caching Hints)
+    -   Accepted by vote of Council on 2021-07-21. (XEP Editor (jsc))
+
+### Deferred
+
+If an experimental XEP is not updated for more than twelve months, it will be moved off Experimental to Deferred. If there is another update, it will put the XEP back onto Experimental.
+
+-   No XEPs deferred this month.
+
+### Updated
+
+-   Version 1.21.0 of [XEP-0060](https://xmpp.org/extensions/xep-0060.html) (Publish-Subscribe)
+    -   Revert change from version 1.15.5 which changed meta-data to metadata in wire protocol. That was an unintended breaking change which has now been reverted. (pep)
+
+-   Version 0.3.0 of [XEP-0214](https://xmpp.org/extensions/xep-0214.html) (File Repository and Sharing)
+    -   Revert change from version 0.2.1 which changed meta-data to metadata in wire protocol. That was an unintended breaking change which has now been reverted. (rm)
+
+-   Version 0.3.0 of [XEP-0248](https://xmpp.org/extensions/xep-0248.html) (PubSub Collection Nodes)
+    -   Revert change from version 0.2.1 which changed meta-data to metadata in wire protocol. That was an unintended breaking change which has now been reverted. (rm)
+
+-   Version 0.2.0 of [XEP-0283](https://xmpp.org/extensions/xep-0283.html) (Moved)
+    -   Re-write the flow with a more focused approach. (mw)
+
+-   Version 1.1.0 of [XEP-0429](https://xmpp.org/extensions/xep-0429.html) (Special Interests Group End to End Encryption)
+    -   Add discussion venue after creation by the Infrastructure Team. (mw)
+
+-   Version 1.24.0 of [XEP-0001](https://xmpp.org/extensions/xep-0001.html) (XMPP Extension Protocols)
+    -   Change "Draft" to "Stable". (ssw)
+
+### Last Call
+
+Last calls are issued once everyone seems satisfied with the current XEP status. After the Council decides whether the XEP seems ready, the XMPP Editor issues a Last Call for comments. The feedback gathered during the Last Call help improving the XEP before returning it to the Council for advancement to Draft.
+
+-   No Last Call this month.
+
+### Draft
```

Comment:
Stable

Reply:
Maybe `### Stable (formely known as Draft)` makes is clearer? And from September it's only 'Stable'

Reply:
Sounds good to me!

Reply:
7c930db